core/services/helper/main.py

```diff
@@ -159,6 +179,35 @@ async def check_internet_access() -> Dict[str, WebsiteStatus]:
         status_list = await asyncio.gather(*tasks)
         return {status.site.name: status for status in status_list}
 
+    @staticmethod
+    def setup_nginx_route(metadata: ServiceMetadata, port: int) -> bool:
+        name: str = re.sub(r"[^a-z0-9]", "", metadata.name.lower())
```

Comment:
We should store ServiceMetadata and avoid running this function to services already registered successfully.